src/openzeppelin/tests/test_erc20.cairo

```diff
@@ -139,33 +164,33 @@ fn test__transfer() {
 
 #[test]
 #[available_gas(2000000)]
-#[should_panic]
+#[should_panic(expected = ('u256_sub Overflow', ))]
```

Comment:
why is `Overflow` uppercase?

Reply:
It's the returned error from the Cairo lib: https://github.com/starkware-libs/cairo/blob/main/corelib/src/integer.cairo#L889